pkg/test/framework/components/cluster/topology.go

```diff
@@ -146,18 +147,27 @@ func (c Topology) MinKubeVersion(major, minor int) bool {
 	if err != nil {
 		return true
 	}
-	serverMajor, err := strconv.Atoi(ver.Major)
+	pv, err := version.ParseGeneric(ver.String())
 	if err != nil {
 		return true
 	}
-	serverMinor, err := strconv.Atoi(ver.Minor)
+	return pv.AtLeast(version.MustParseGeneric(fmt.Sprintf("v%d.%d.0", major, minor)))
+}
+
+func (c Topology) MaxKubeVersion(minor uint) bool {
```

Comment:
nit: should this take major, minor like the Min version?

Reply:
they should probably consistently use min. k8s will never change major version

Reply:
really? There will *never* be a kubernetes 2.0?

Reply:
Not while I am around :stuck_out_tongue: . I think they have said they have no plans to ever have one, nor do we.